playbooks/roles/aws_devstack/tasks/main.yml

```diff
@@ -37,7 +37,7 @@
       cd edx-repos/devstack/
       make dev.pull.large-and-slow
       make dev.up.lms
-      . /home/ubuntu/.profile && docker compose exec lms env TERM=xterm-256color bash -c 'make requirements && npm ci && paver update_assets && python manage.py lms migrate'
+      . /home/ubuntu/.profile && docker compose exec lms env TERM=xterm-256color bash -c 'make requirements && npm ci && npm run build && ./manage.py lms collectstatic --noinput && ./manage.py cms collectstatic && python manage.py lms migrate'
```

Comment:
Do we need to add ``--noinput`` argument for ``cms collectstatic`` command as well?

Reply:
Just used the same parameter for cms to keep the standard same. Should we not add it to standardize command?

Reply:
For reference, I followed Kyle's suggested replacement for `paver deprecation` so used `--no-input` only once.